lib/tapioca/compilers/dsl/smart_properties.rb

```diff
@@ -76,7 +76,7 @@ def decorate(root, constant)
           root.create_path(constant) do |k|
             properties.values.each do |property|
               generate_methods_for_property(k, property) do |method_name|
-                !instance_methods.include?(method_name.to_sym)
+                !instance_methods.include?(method_name)
```

Comment:
`instance_methods` is a set of Strings created using `to_set` a few[ lines above](https://github.com/Shopify/tapioca/pull/673/files#diff-835bd084a3e5ca4cfda22fb33c1f2a5d77ed708b1510958adbd33789cc3c105bR74). `to_set` [no longer returns `T.untyped`](https://github.com/sorbet/sorbet/commit/5d3cc153ec60bb2c652b44a47b0f78288e36b631), making using symbols a typecheck error.